addons/website_event/models/event_event.py

```diff
@@ -42,6 +42,25 @@ def _default_cover_properties(self):
     menu_register_cta = fields.Boolean(
         'Add Register Button', compute='_compute_menu_register_cta',
         readonly=False, store=True)
+    # sub-menus management
+    introduction_menu = fields.Boolean(
```

Comment:
Is the help attribute of "website_menu" still accurate?
```py
help="Creates menus Introduction, Location and Register on the page "
             "of the event on the website."
```

Side-note, was wondering if we should not prefix all menu-things with menu to have some namespacing.

So menu_introduction, menu_introduction_ids, ...
But maybe it would be a bit annoying to do and not aligned with existing stuff.

Reply:
Then we would have to rename all menus fields (we have quite a lot of them: track, rooms, exhibitors, twitter wall, ...) . Not sure, I always hesitated between "track_menu" and "menu_track" . Maybe for a next cleaning task :D .

Reply:
Docstring updated.